### ISSUE TITLE
Solution for problem 67 incorrect

### ISSUE THREAD

**Issue by thisisnic** [nathaneastwood/euler#9]
When I enter the result of calling `euler_67(data_67)` (7272) into the answer box at https://projecteuler.net/problem=67, the result is flagged as incorrect.

**Comment by nathaneastwood** [comment]
Hmm that's odd. If I give the function the data from the example on the website I get the correct solution:

```r
data <- structure(c(3, 7, 2, 8, 0, 4, 4, 5, 0, 0, 6, 9, 0, 0, 0, 3), .Dim = c(4L, 4L))
euler_67(data)
# [1] 23
```

`euler_67(data_67)` gives 7272 but other solutions online are giving 7273...

**Comment by nathaneastwood** [comment]
I'm not why, but the first number was a `58` and not `59`. I must have somehow overwritten it...now fixed as of 24697e5.
